src/onetime/entrypoints/web/Flask/views.py

```diff
@@ -0,0 +1,47 @@
+
+from flask import request, url_for, render_template, session
+from .app import app
+from .form import SecretCreateForm
+from dependency_injector.wiring import Provide, inject
+from onetime.configurator.containers import Container
+from onetime.use_cases.manager import SecretAndUrlManager
+
+#exceptions
+from onetime.use_cases.exceptions import (
+    SecretDataWasAlreadyConsumedException,
+    URLExpiredException,
+    UUIDNotFoundException,
+)
+
+
+@app.route('/', methods=['GET', 'POST'])
+@inject
+def index():
+    form = SecretCreateForm()
+    if request.method == "POST":
+        form = SecretCreateForm(request.form)
+        if not form.validate_on_submit():
+            container = Container()
```

Comment:
The web view must not be responsible for the DI container setup.
The dependencies must be already available beforehand for the view.
Please see Django application for the proper DI usage.

Reply:
solved.

A new container was written for flask.

in another, get_secret and other methods were not visible.